ui_framework/src/test/snapshot_component.js

```diff
@@ -0,0 +1,6 @@
+export const snapshotComponent = component => {
```

Comment:
Can you add some docs on when this should be used? Looks like it's only for `mount`ed components, for example.